pkg/cli/cmds/agent.go

```diff
@@ -54,8 +54,16 @@ func NewAgentCommand() cli.Command {
 }
 
 func AgentRun(clx *cli.Context) error {
-	if profile == "" {
+	switch profile {
```

Comment:
This would probably make sense to have in a function in root.go that takes the profile string as an arg and does this logic so it's not needed to be maintained in 2 different places. More easily extensible too if/when we move to 1.6, etc. Not blocking however it'd be cool to have an issue that captures this.

Reply:
I agree 100%. I had originally moved this into the validateCISreqs function, but Craig asked me to reduce the amount of change so I went for a more minimal approach at the cost of some repetition.

Reply:
+1. He basically had that, but I asked him to minimize the pr to the smallest change possible for the purpose of reviewing and merging as quickly as possible.

Reply:
Cool. Do we have an issue for this for after GA?